app/services/events.py

```diff
@@ -0,0 +1,38 @@
+import json
+import logging
+from typing import Dict
+
+
+class EventsService:
+
+    @classmethod
```

Comment:
We shouldn't use `classmethod` and `staticmethod` for the services, they should be instantiated

Reply:
Updated [here](https://github.com/safe-global/safe-decoder-service/pull/24/commits/127b804838ce17deaeea9c8e1f4e7ae99de78ea1)